script.js

```diff
@@ -26,40 +30,50 @@ var rollDice = function () {
 var playerRoll = function () {
   playerRolls.push(rollDice());
   playerRolls.push(rollDice());
+  console.log(`playerRolls: ${playerRolls}`);
   myOutputValue = `Welcome Player ${playerTurn}. <br> Dice 1: ${playerRolls[0]} <br> Dice 2: ${playerRolls[1]} <br> Please choose the order of the dice.`;
   return myOutputValue;
 };
 
 var main = function (input) {
```

Comment:
I would recommend starting to move a majority of your code outside the var main function, and instead create a bunch of helper functions outside the var main.  For example in this situation, you could move the majority of the code outside the var main and have a function that encompasses all the game logic.
```
var gameLogic= function (input) {
... // code here
}

var main = function (input) {
  return gameLogic(input);
};
```
Helps with readability and if you ever want to use the gameLogic outside of the var main function you'll be able to do that.